Increase the timeout of integration tests

### General information

CI build getting timeout since now the integration tests are taking more than 60mins. 
However, the default `TIMEOUT` set for running integration test is `3600s` or `60mins`.

https://github.com/minishift/minishift/blob/05207b3d9dcdd4d01f1d47e867de9f7d909bd6ca/Makefile#L35

### Expected

Integration tests shouldn't be failed on 60mins. 

### Actual

Integration tests are getting timeout on 60mins.

### Logs

See https://ci.centos.org/job/minishift-release/77/consoleFull

```
01:06:42.047 *** Test killed with quit: ran too long (1h1m0s).
01:06:42.047 FAIL	github.com/minishift/minishift/test/integration	3660.051s
01:06:42.047 make: *** [integration_all] Error 1
01:06:42.058 Connection to 172.19.3.88 closed.
```

--- a/Makefile
+++ b/Makefile
@@ -32,7 +32,7 @@ ifeq ($(GOOS),windows)
 	IS_EXE := .exe
 endif
 MINISHIFT_BINARY ?= $(GOPATH)/bin/minishift$(IS_EXE)
-TIMEOUT ?= 3600s
+TIMEOUT ?= 7200s
 PACKAGES := go list ./... | grep -v /vendor | grep -v /out
 SOURCE_DIRS = cmd pkg test
 

